aws/app-lb/main.tf

```diff
@@ -0,0 +1,216 @@
+## Get Private subnets from VPC
+data "aws_subnets" "private" {
```

Comment:
yes, I never mentioned about the subnets: The external ALB should be attached to the public subnets while the internal LB should be attached to the private ones.  This is not a hard rule, but the way we design our network, at least for now